Crash when using --db-only flag

```
issue 379 already exists - ignored
issue 380 already exists - ignored
committing transaction
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x1004930a4]

goroutine 1 [running]:
gopkg.in/src-d/go-git%2ev4.(*Repository).Remote(0x0, {0x1005cfac1, 0x6})
	/Users/xxx/go/pkg/mod/gopkg.in/src-d/go-git.v4@v4.13.1/repository.go:444 +0x24
gopkg.in/src-d/go-git%2ev4.(*Repository).PushContext(0x0?, {0x10079e978, 0x140001a4028}, 0x14000291d48)
	/Users/xxx/go/pkg/mod/gopkg.in/src-d/go-git.v4@v4.13.1/repository.go:1037 +0x48
gopkg.in/src-d/go-git%2ev4.(*Repository).Push(...)
	/Users/xxx/go/pkg/mod/gopkg.in/src-d/go-git.v4@v4.13.1/repository.go:1022
github.com/stevejefferson/trac2gitea/accessor/gitea.(*DefaultAccessor).commitWikiRepo(0x140001fe1b0)
	/Users/xxx/Source/trac2gitea/accessor/gitea/wiki.go:234 +0x118
github.com/stevejefferson/trac2gitea/accessor/gitea.(*DefaultAccessor).CommitTransaction(0x140001fe1b0)
	/Users/xxx/Source/trac2gitea/accessor/gitea/transaction.go:14 +0x40
github.com/stevejefferson/trac2gitea/importer.(*Importer).CommitImport(0x14000236740)
	/Users/xxx/Source/trac2gitea/importer/transaction.go:14 +0x58
main.performImport(0x16fd537fd?, 0x1e?, 0x140000546c8?, 0x140000021a0?, 0x14000054708?, 0x10010f568?, 0x140000021a0?, 0x300000002?, 0x140001e7f28?)
	/Users/xxx/Source/trac2gitea/main.go:175 +0x5c
main.main()
	/Users/xxx/Source/trac2gitea/main.go:252 +0x37c
```

NOTE: The DB changes are actually committed.

--- a/accessor/gitea/defaultAccessor.go
+++ b/accessor/gitea/defaultAccessor.go
@@ -36,6 +36,7 @@ type DefaultAccessor struct {
 	wikiRepo      *git.Repository
 	overwrite     bool
 	pushWiki      bool
+	dbOnly        bool
 }
 
 func fetchConfig(configPath string) (*ini.File, error) {
@@ -64,7 +65,8 @@ func CreateDefaultAccessor(
 	giteaWikiRepoToken string,
 	giteaWikiRepoDir string,
 	overwriteData bool,
-	pushWiki bool) (*DefaultAccessor, error) {
+	pushWiki bool,
+	dbOnly bool) (*DefaultAccessor, error) {
 	stat, err := os.Stat(giteaRootDir)
 	if err != nil {
 		err = errors.Wrapf(err, "looking for root directory %s of Gitea instance", giteaRootDir)
@@ -125,6 +127,7 @@ func CreateDefaultAccessor(
 		wikiRepo:      nil,
 		overwrite:     overwriteData,
 		pushWiki:      pushWiki,
+		dbOnly:        dbOnly,
 	}
 
 	dialect, dbType, err := giteaAccessor.getDbDialect()

--- a/accessor/gitea/transaction.go
+++ b/accessor/gitea/transaction.go
@@ -11,7 +11,11 @@ func (accessor *DefaultAccessor) CommitTransaction() error {
 		return err
 	}
 
-	return accessor.commitWikiRepo()
+	if !accessor.dbOnly {
+		return accessor.commitWikiRepo()
+	}
+
+	return nil
 }
 
 // RollbackTransaction rolls back a Gitea transaction.
@@ -21,5 +25,9 @@ func (accessor *DefaultAccessor) RollbackTransaction() error {
 		return err
 	}
 
-	return accessor.rollbackWikiRepo()
+	if !accessor.dbOnly {
+		return accessor.rollbackWikiRepo()
+	}
+
+	return nil
 }

--- a/main.go
+++ b/main.go
@@ -187,7 +187,7 @@ func createImporter() (*importer.Importer, error) {
 		return nil, err
 	}
 	giteaAccessor, err := gitea.CreateDefaultAccessor(
-		giteaRootDir, giteaMainConfigPath, giteaOrg, giteaRepo, giteaWikiRepoURL, giteaWikiRepoToken, giteaWikiRepoDir, overwrite, wikiPush)
+		giteaRootDir, giteaMainConfigPath, giteaOrg, giteaRepo, giteaWikiRepoURL, giteaWikiRepoToken, giteaWikiRepoDir, overwrite, wikiPush, dbOnly)
 	if err != nil {
 		return nil, err
 	}

